microsoft-365/compliance/create-custom-sensitive-information-types-with-exact-data-match-based-classification.md

```diff
@@ -87,21 +87,22 @@ EDM-based classification is included in these subscriptions
+The data file can include a maximum of:
       - Up to 100 million rows of sensitive data
       - Up to 32 columns (fields) per data source
       - Up to 5 columns (fields) marked as searchable
 
-2. Structure the sensitive data in the .csv file such that the first row includes the names of the fields used for EDM-based classification. In your .csv file, you might have field names, such as "ssn", "birthdate", "firstname", "lastname". The column header names can't include spaces or underscores. For example, the sample .csv file that we use in this article is named *PatientRecords.csv*, and its columns include *PatientID*, *MRN*, *LastName*, *FirstName*, *SSN*, and more.
+2. Structure the sensitive data in the .csv or .tsv file such that the first row includes the names of the fields used for EDM-based classification. In your file you might have field names such as "ssn", "birthdate", "firstname", "lastname". The column header names can't include spaces or underscores. For example, the sample .csv file that we use in this article is named *PatientRecords.csv*, and its columns include *PatientID*, *MRN*, *LastName*, *FirstName*, *SSN*, and more.
 
-3. Pay attention to the format of the sensitive data fields. In particular, fields that may contain commas in their content (e.g. a street address that contains the value "Seattle,WA") would be parsed as two separate fields when parsed by the EDM tool. In order to avoid this, you need to ensure such fields are surrounded by single or double quotes in the sensitive data table. If fields with commas in them may also contain spaces, you would need to create a custom Sensitive Information Type that matches the corresponding format (e.g. a multi-word string with commas and spaces in it) to ensure the string is correctly matched when the document is scanned.
+3. Pay attention to the format of the sensitive data fields. In particular, fields that may contain commas in their content (e.g. a street address that contains the value "Seattle,WA") would be parsed as two separate fields when parsed by the EDM tool if the .csv format is selected. In order to avoid this, either use a file with the .tsv format or ensure such fields are surrounded by single or double quotes in the sensitive data table (double quotes are generally preferred since single quotes may be confused with apostrophes in names and words in certain languages). If fields with commas in them may also contain spaces, you would need to create a custom Sensitive Information Type that matches the corresponding format (e.g. a multi-word string with commas and spaces in it) to ensure the string is correctly matched when the document is scanned.
```

Comment:
```suggestion
3. Pay attention to the format of the sensitive data fields. In particular, fields that may contain commas in their content (for example, a street address that contains the value "Seattle,WA") would be parsed as two separate fields when parsed by the EDM tool if the .csv format is selected. In order to avoid this, either use a file with the .tsv format or ensure such fields are surrounded by single or double quotes in the sensitive data table (double quotes are generally preferred since single quotes may be confused with apostrophes in names and words in certain languages). If fields with commas in them also contain spaces, you would need to create a custom Sensitive Information Type that matches the corresponding format (for example, a multi-word string with commas and spaces in it) to ensure the string is correctly matched when the document is scanned.
```